index.html

```diff
@@ -9,5 +9,6 @@
     <h1>BJET Task</h1>
     <h1>BJET Task 2</h1>
     <h1>BJET Task 3</h1>
+    <p>This is a sample HTML paragraph.The <p> tags define a paragraph, and the text inside them is the content of the paragraph.</p>
```

Comment:
@TarannaumSultan san You use two p tags but you didn't close the first tag
Please close the first p tag.